Sources/PayPal/Enums/PayPalError.swift

```diff
@@ -0,0 +1,28 @@
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+#if canImport(PayPalCheckout)
+import PayPalCheckout
+#endif
```

Comment:
I love this!

Reply:
Yeah let's test this! If nothing else, it will help enforce a separation between NXO interfaces and Northstar interfaces, since it will throw a compile time error if you try to expose an NXO type while using an `_implementationOnly` import